src/utils/constants/constants.js

```diff
@@ -0,0 +1,25 @@
+import markIcon from "src/assets/images/mark.png";
+import penIcon from "src/assets/images/pen.png";
+import binIcon from "src/assets/images/bin.png";
+import logo from "src/assets/images/logo.png";
+import navLogo from "src/assets/images/nav-logo.png";
+import searchIcon from "src/assets/images/search.png";
+import addIcon from "src/assets/images/add.png";
+
+export const ALT_MARK_TEXT = "Mark task as done";
+export const ALT_EDIT_TEXT = "Edit task";
+export const ALT_DELETE_TEXT = "Delete task";
+export const ALT_LOGO_TEXT = "Logo";
+export const ALT_SEARCH_TEXT = "Search";
+export const ALT_NAV_LOGO_TEXT = "Logo";
+export const ALT_ADD_TEXT = "Add Task Button";
+export const TEXT_CREATE = "create";
+export const CLASS_BUTTON_CREATE = "content__button--create";
+
+export const ICON_MARK = markIcon;
+export const ICON_PEN = penIcon;
+export const ICON_BIN = binIcon;
+export const LOGO = logo;
+export const NAV_LOGO = navLogo;
+export const ICON_SEARCH = searchIcon;
+export const ICON_ADD = addIcon;
```

Comment:
Move these to a different file as they seems identical. 

It is a good idea to breaks into module based to characters. It will give more modular experience.